src/js/common/utils/lodash-replacements/get.js

```diff
@@ -0,0 +1,26 @@
+import deconstructPath from './deconstructPath';
+
+/**
+ * Gets a the value at the end of the path.
+ *
+ * @param {Object} object
+ * @param {Array|string} path
+ * @param {*} [defaultValue]
+ * @return {*}
+ */
+export default function get(path, object, defaultValue) {
+  const arrayPath = Array.isArray(path) ? path : deconstructPath(path);
+  let currentValue = object;
+
+  for (let i = 0; i < arrayPath.length; i++) {
+    if (typeof currentValue[arrayPath[i]] !== 'undefined') {
+      currentValue = currentValue[arrayPath[i]];
+    } else {
+      return defaultValue;
+    }
+  }
+
+  // Should this clone? the current value? It might use a different ref--not sure.
```

Comment:
I think not cloning is fine.

Reply:
Without cloning, modifying the result of `_.get()` modifies the original:
```js
const obj = {
  sub: {
    foo: 'bar'
  }
};

const sub = _.get(['sub'], obj);
sub.foo = 'baz';

console.log(obj.sub.foo); // output: baz
```
Our options are
1. `cloneDeep` the return value to make it safe
2. Put a comment in there to note that modifying the return value isn't safe